src/Integration.Vsix/AsmRef_Integration.Vsix_Baseline_WithStrongNames.txt

```diff
@@ -282,7 +282,8 @@ Referenced assemblies:
 - 'System.ComponentModel.Composition, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089'
 - 'System.Core, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089'
 - 'System.Xaml, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089'
-# Number of references: 22
+- 'WindowsBase, Version=4.0.0.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35'
```

Comment:
I don't understand where this is suddenly coming from, and why I don't see this on the main branch-taint.
It was always there...
https://github.com/SonarSource/sonarlint-visualstudio/blob/master/src/IssueViz.Security/IssueViz.Security.csproj#L41

Reply:
Maybe it wasn't actually needed before so VS wasn't including the reference in the assembly?

Have you checked whether the assembly actually contains a reference to WindowsBase (using e.g. `ildasm`)?

I have a similar problem with my latest changes: I'm seeing `- 'System, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089'` appearing as a reference for `SonarLint.VisualStudio.Infrastructure.VS`, and I haven't made any changes to that assembly and it doesn't depend on anything I have changed. I haven't investigated using `ildasm` yet.

Reply:
Update: my build problem turned out to be due a difference between Debug and Release builds (building locally means the Infra assembly includes Debug.Assert calls, building on the server means it doesn't, and presumably the code doesn't reference anything else from that assembly).

I wouldn't expect it to make a difference here, but you might want to try it.

Reply:
Seems like it indeed wasn't needed before - it's used now for `CollectionViewSource.GetDefaultView`.  
I don't know why it was there before then (in the csproj) -- I guess it was not needed anymore at some point but the csproj was not updated.